modules/vstudio/_preload.lua

```diff
@@ -195,6 +195,48 @@
 		}
 	}
 
+	p.api.register {
+		name = "toolchainversion",
+		scope = "config",
+		kind = "string",
+		allowed = {}
+	}
+
+--
+-- Register Linux properties
+--
+
+	p.api.addAllowed("toolchainversion", { "remote", "wsl", "wsl2" })
+
+	-- Directory in the remote machine where our files will be copied before compilation
+	p.api.register {
+		name = "remoterootdir",
+		scope = "config",
+		kind = "string",
```

Comment:
```suggestion
		kind = "path",
```
?
Apply to other added api.

Reply:
Aha, I wasn't aware we had thing kind of thing, will do and test. Is it because path is able to resolve tokens?

Reply:
Hmm unfortunately it tries to form a path on the local machine, so for example something like `~/projects/crstl` turns into `C:/Users/redorav/Desktop/crstl/~/projects/crstl` and that's not what we want

Reply:
ok